scout/server/blueprints/cases/views.py

```diff
@@ -1088,7 +1088,7 @@ def mark_causative(institute_id, case_name, variant_id, partial_causative=False)
         else:
             store.mark_causative(institute_obj, case_obj, user_obj, link, variant_obj)
     elif request.form["action"] == "DELETE":
-        if partial_causative:
+        if partial_causative == "True":
```

Comment:
It's passed as an arg, so it's a string

Reply:
Ouch..

Reply:
Thanks, sorry - that was likely from my CodeFactor fix drive..

Reply:
Ah, no, wasn't but anyway.

Reply:
😆 Was indeed - the removed eval! Thanks!

Reply:
What?

Reply:
Ah Ok. Whatever! 😄